app.go

```diff
@@ -134,6 +134,58 @@ func (c *Client) UpdateAppSettings(ctx context.Context, settings *AppSettings) (
 	return &resp, err
 }
 
+type CheckSqsRequest struct {
```

Comment:
Let's make SQS capital letters.
```suggestion
type CheckSQSRequest struct {
```

Reply:
in all occurences